sdk/python/jobs/automl-standalone-jobs/automl-distributed-classification-task-creditcard/classification-distributed-lightgbm.ipynb

```diff
@@ -0,0 +1,1472 @@
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "from azure.core.exceptions import ResourceNotFoundError\n",
+    "from azure.ai.ml.entities import AmlCompute\n",
+    "\n",
+    "cluster_name = \"automl-distributed-cluster\"\n",
+    "\n",
+    "try:\n",
+    "    # Retrieve an already attached Azure Machine Learning Compute.\n",
+    "    compute = ml_client.compute.get(cluster_name)\n",
+    "except ResourceNotFoundError as e:\n",
+    "    compute = AmlCompute(\n",
+    "        name=cluster_name,\n",
+    "        size=\"STANDARD_DS12_V2\",\n",
+    "        type=\"amlcompute\",\n",
+    "        min_instances=4,\n",
+    "        max_instances=6,\n",
+    "        idle_time_before_scale_down=120,\n",
+    "    )\n",
+    "    poller = ml_client.begin_create_or_update(compute)\n",
+    "    poller.wait()"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1684844847439
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 4. Configure and run the Distributed Classification training job\r\n",
+    "\r\n",
+    "In this section we will configure and run the AutoML job to train the model.\r\n",
+    "\r\n",
+    "#### 4.1 Configure the job through the classification() factory function\r\n",
+    "\r\n",
+    "##### classification() function parameters:\r\n",
+    "\r\n",
+    "The `classification()` factory function allows user to configure AutoML for the classification task for the most common scenarios with the following properties.\r\n",
+    "\r\n",
+    "|Property|Description|\r\n",
+    "|-|-|\r\n",
+    "|**target_column_name**|The name of the label column.|\r\n",
+    "|**primary_metric**|This is the metric that you want to optimize. In case this parameter is set to `None`, `accuaracy` is used.|\r\n",
+    "|**training_data**|The training data to be used for this experiment. You can use a registered MLTable in the workspace using the format `<mltable_name>:<version>` OR you can use a local file or folder as a MLTable. For e.g `Input(mltable='my_mltable:1')` OR `Input(mltable=MLTable(local_path=\"./data\"))` The parameter 'training_data' must always be provided.|\r\n",
+    "|**compute**|The compute on which the AutoML job will run. In this example we are using a compute called 'cpu-cluster' present in the workspace. You can replace it with any other compute in the workspace.|\r\n",
```

Comment:
compute name is `automl-distributed-cluster`. I recommend to have a unique name for the compute. If regression and classification notebooks use the same compute name there will be failed notebook runs.